subsys/bluetooth/services/ots/Kconfig

```diff
@@ -13,14 +13,34 @@ config BT_OTS
 
 if BT_OTS
 
+config BT_OTS_DIR_LIST_OBJ
+	bool "Enables the Directory Listing Object"
+	help
+	  Enables the Directory Listing Object, which is an object that contains all the metadata
+	  from all other objects, for easy exposure to a client. Enabling this will use one of the
+	  objects given by BT_OTS_MAX_OBJ_CNT.
+
+config BT_OTS_DIR_LIST_OBJ_NAME
+	string "The object name of the Directory Listing Object"
+	default "Directory"
+	depends on BT_OTS_DIR_LIST_OBJ
+	help
+	  The name of the Directory Listing Object when it is read by a client.
+
 config BT_OTS_MAX_INST_CNT
 	int "Maximum number of available OTS instances"
 	default 1
 	range 1 1 if !BT_OTS_SECONDARY_SVC
 
 config BT_OTS_MAX_OBJ_CNT
-	int "Maximum number of objects that each service instance can store"
-	default 5
+	hex "Maximum number of objects that each service instance can store"
```

Comment:
Why `hex` instead of `int`? Usually it's more natural to represent "a number of something" in decimal rather than hexadecimal.

Reply:
I agree. I changed it to make the ranges easier to compare to the specification. 

Depending on the BT_OTS_DIR_LIST_OBJ, there are 2 max ranges, and writing `0xFFFFFFFFFFFFFEFF` in decimal seemed wrong to me, but I can change it back again.